templates/vue/src/components/NodeModal.vue

```diff
@@ -1,74 +1,107 @@
 <template>
-  <b-modal id="node-modal-container" size="lg" class="text-muted" scrollable :title="modalTitle" body-class="p-0">
+  <b-modal
+    id="node-modal-container"
+    size="lg"
+    class="text-muted"
+    scrollable
+    :title="modalTitle"
+    body-class="p-0"
+  >
     <b-container fluid class="px-0">
       <b-tabs card>
         <b-tab title="Content" active>
           <div id="modal-content-details">
-            <b-alert v-if="formErrors.length" variant="danger" id="tapestry-modal-form-errors" v-html="formErrors" show>
+            <b-alert
+              v-if="formErrors.length"
+              id="tapestry-modal-form-errors"
+              variant="danger"
+              show
+              v-html="formErrors"
+            >
             </b-alert>
             <b-form-group label="Title">
               <b-form-input
                 id="node-title"
-                placeholder="Enter title"
                 v-model="node.title"
+                placeholder="Enter title"
                 autofocus
                 required
               />
             </b-form-group>
             <b-form-group label="Description">
               <b-form-textarea
                 id="node-description"
-                placeholder="Enter description"
                 v-model="node.description"
+                placeholder="Enter description"
               ></b-form-textarea>
             </b-form-group>
             <b-form-group label="Content Type">
-              <b-form-select id="node-media-type" v-model="node.mediaType" :options="mediaTypes"></b-form-select>
+              <b-form-select
+                id="node-media-type"
+                v-model="node.mediaType"
+                :options="mediaTypes"
+              ></b-form-select>
             </b-form-group>
-            <b-form-group label="Text content" v-show="node.mediaType === 'text'">
+            <b-form-group
+              v-show="node.mediaType === 'text'"
+              label="Text content"
+            >
               <b-form-textarea
                 id="node-text-content"
-                placeholder="Enter text here"
                 v-model="node.typeData.textContent"
+                placeholder="Enter text here"
               ></b-form-textarea>
             </b-form-group>
-            <b-form-group label="Video URL" v-show="node.mediaType === 'video'">
+            <b-form-group v-show="node.mediaType === 'video'" label="Video URL">
               <b-form-input
                 id="node-video-media-url"
-                placeholder="Enter URL for MP4 Video"
                 v-model="node.typeData.mediaURL"
+                placeholder="Enter URL for MP4 Video"
                 required
               />
             </b-form-group>
-            <b-form-group label="Video Duration" v-show="node.mediaType === 'video'">
+            <b-form-group
+              v-show="node.mediaType === 'video'"
+              label="Video Duration"
+            >
```

Comment:
How come the first b-form-group element is in one line and the second one is in 2 lines? Anyway to keep this consistent? I'm guessing it's due to the length of the line, but I am still curious to see if it's possible to keep it consistent somehow.